docs/devcontainer-spec-support.md

```diff
@@ -0,0 +1,72 @@
+- 🔴 Not currently supported.
+
+The last column indicates any currently existing GitHub issue for tracking support for this feature.
+Feel free to [create a new issue](https://github.com/coder/envbuilder/issues/new) if you'd like Envbuilder to support a particular feature.
+
+| Status                        | Name                                  | Description                                                                                                            | Known Issues                                           |
+| ----------------------------- | ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| **General**                   |                                       |                                                                                                                        |
+| 🔴                            | `name`                                | Human-friendly name for the dev container.                                                                             | -                                                      |
+| 🔴                            | `forwardPorts`                        | Port forwarding allows exposing container ports to the host, making services accessible.                               | [#48](https://github.com/coder/envbuilder/issues/48)   |
+| 🔴                            | `portsAttributes`                     | Set port attributes for a `host:port`.                                                                                 | -                                                      |
+| 🔴                            | `otherPortsAttributes`                | Other options for ports not configured using `portsAttributes`.                                                        | -                                                      |
+| 🟢                            | `containerEnv`                        | Environment variables to set inside the container.                                                                     | -                                                      |
+| 🟢                            | `remoteEnv`                           | Override environment variables for specific supporting services or tools, but not the container as a whole.            | -                                                      |
+| 🟢                            | `remoteUser`                          | Override the user that supporting tools run as inside the container without changing the user for the whole container. | -                                                      |
+| 🟢                            | `containerUser`                       | Override the user for all operations run inside the container.                                                         | -                                                      |
```

Comment:
With these, we may have some divergence from the spec. I think we can leave it green but we may need to verify/disclaimer.

Reply:
I'm finding it difficult to define exactly how we diverge from the spec here. As far as I can tell, we just append `USER $_REMOTE_USER` after the feature directives but we don't appear to use `remoteUser` for anything else.